kolibri/core/assets/src/utils/branding.js

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue';
+
+const schema = {
+  appBar: {
+    topLogo: {
+      src: null,
+      alt: null,
+      style: null,
+    },
+  },
+  signIn: {
+    topLogo: {
+      src: null,
+      alt: null,
+      style: null,
+    },
+    title: null,
+    showTitle: null,
+    titleStyle: null,
+    showPoweredBy: null,
+    poweredByStyle: null,
+    showKolibriFooterLogo: null,
+    background: null,
+  },
+  sideNav: {
+    topLogo: {
+      src: null,
+      alt: null,
+      style: null,
+    },
+    showKolibriFooterLogo: null,
+  },
+};
```

Comment:
cool

Reply:
I feel like I have a better idea of what the API for this now is too!